scripts/convert_iob_format_to_token_classification_format.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.****
 
 import argparse
+import logging
```

Comment:
this should be nemo.logging, no?

Reply:
I left all scripts as import logging as technically, scripts should be able to be run without installing nemo